servlet-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/http-client/common/main/module.xml

```diff
@@ -25,7 +25,7 @@
 <module xmlns="urn:jboss:module:1.9" name="org.wildfly.http-client.common">
 
     <resources>
-        <artifact name="${org.wildfly.wildfly-http-client:wildfly-http-client-common}"/>
+         <artifact name="\${org.wildfly.wildfly-http-client:wildfly-http-client-common@module.jakarta.suffix@}"/>
```

Comment:
You can just make this:

${org.wildfly.wildfly-http-client:wildfly-http-client-common-jakarta}